test/e2e/common.go

```diff
@@ -27,6 +28,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type vmState string
+
+const (
+	running vmState = "running"
+	paused  vmState = "paused"
+	shutoff vmState = "shutoff"
+	other   vmState = "other"
+)
```

Comment:
I would keep them in their own e2e test file if possible unless there is something I am missing here or at least merge others consts (lines 104-107) in this file into one

Reply:
merged